### PR TITLE
we were checking if the parent of the source node was in view, now checking if the source node is in view of the tree pane

### PR DESCRIPTION
**Pull request by karina-y**

### CODE REVIEW

**Comment by karina-y** [comment]
ran Test Stripe 2's failed test, inspector-01, locally and all tests passed
ran Test Stripe 3's failed test, logpoint-03, locally and all tests passed
ran Test Stripe 5's failed test, object_preview-01, locally and all tests passed
ran Test Stripe 6's failed test, console_eval, locally and all tests passed

going to rerun all jobs in ci and see if they pass a second time around

File: src/devtools/client/debugger/dist/vendors.js [hunk 1]
```diff
@@ -1017,7 +1017,8 @@
                   return closestScrolledParent(node.parentNode);
                 };
 
-                const scrolledParent = closestScrolledParent(treeElement);
+                //check if our element is currently in view of the tree pane
+                const scrolledParent = treeElement;
```
[the remark this answers is not in the document]
Comment (jasonLaster) [reply src/devtools/client/debugger/dist/vendors.js:1021]: One thought is that making the change in vendors is super scary. Lets see if we can make the change in the original src file and then just re-route the imports

```diff
diff --git a/src/devtools/client/debugger/dist/devtools-components.js b/src/devtools/client/debugger/dist/devtools-components.js
index 00d59ef2..81aaa259 100644
--- a/src/devtools/client/debugger/dist/devtools-components.js
+++ b/src/devtools/client/debugger/dist/devtools-components.js
@@ -1,2 +1 @@
-const { vendored } = require("./vendors");
-module.exports = vendored["devtools-components"];
+export * from "devtools/client/debugger/packages/devtools-components";
diff --git a/src/devtools/client/debugger/packages/devtools-components/index.js b/src/devtools/client/debugger/packages/devtools-components/index.js
index 1ae316ef..cbeab6b5 100644
--- a/src/devtools/client/debugger/packages/devtools-components/index.js
+++ b/src/devtools/client/debugger/packages/devtools-components/index.js
@@ -2,8 +2,4 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */

-import Tree from "./src/tree";
-
-module.exports = {
-  Tree,
-};
+export { default as Tree } from "./src/tree";
diff --git a/src/devtools/client/debugger/packages/devtools-reps/src/object-inspector/components/ObjectInspector.js b/src/devtools/client/debugger/packages/devtools-reps/src/object-inspector/components/ObjectInspector.js
index de9b33b5..e12ccda5 100644
--- a/src/devtools/client/debugger/packages/devtools-reps/src/object-inspector/components/ObjectInspector.js
+++ b/src/devtools/client/debugger/packages/devtools-reps/src/object-inspector/components/ObjectInspector.js
@@ -10,8 +10,9 @@ const actions = require("../actions");

 const selectors = require("../reducer");

-import Components from "devtools-components";
-const Tree = createFactory(Components.Tree);
+import { Tree as _Tree } from "devtools-components";
+
+const Tree = createFactory(_Tree);
 require("./ObjectInspector.css");

 const ObjectInspectorItem = createFactory(require("./ObjectInspectorItem").default);
(END)
```

P.S. Please confirm that the code is the same. It shouldn't have changed, but who knows

**Review by jasonLaster — COMMENTED** [review]
Looks good. Will test tomorrow.

**Comment by jasonLaster** [comment]
Which issue does this fix? we're fixing too many to be sure :)

**Comment by karina-y** [comment]
@jasonLaster all failed tests in ci pass locally except breakpoints-06 which times out

**Comment by karina-y** [comment]
this pr fixes the import, using component instead of vendors. related to issue #1173

**Pull request merged**